VAMobile/src/components/FormWrapper/FormFields/VATextInput.test.tsx

```diff
@@ -1,113 +1,74 @@
 import 'react-native'
 import React from 'react'
-import { TextInput } from 'react-native'
-// Note: test renderer must be required after react-native.
-import 'jest-styled-components'
-import { ReactTestInstance } from 'react-test-renderer'
-import { screen } from '@testing-library/react-native'
-import Mock = jest.Mock
-
-import { context, render, RenderAPI, waitFor } from 'testUtils'
-import VATextInput, { VATextInputTypes } from './VATextInput'
-import { isIOS } from 'utils/platform'
-
-let mockIsIOS = jest.fn()
-jest.mock('utils/platform', () => ({
-  isIOS: jest.fn(() => mockIsIOS),
-}))
+import { context, render, screen, fireEvent } from 'testUtils'
+import VATextInput from './VATextInput'
 
 context('VATextInput', () => {
-  let component: RenderAPI
-  let testInstance: ReactTestInstance
-  let onChangeSpy: Mock
-  let isIOSMock = isIOS as jest.Mock
-
-  const initializeTestInstance = (
```

Comment:
I would keep initializeTestInstance, except reduce the number of arguments down to just what needs to change for the specific tests you have. That way you only have one place where you call `render`, and it's as simple as possible

Reply:
I thought I was supposed to get rid of `testInstance` all together?

Reply:
Definitely get rid of `testInstance` itself, that's not needed. However the pattern of rendering the component in just one place is still good. I prefer the name `renderWithProps` instead of `initializeTestInstance`, but we don't really have a standard for how to name that function

Reply:
Updated.